README.md

```diff
@@ -368,6 +368,7 @@ And thus it won't be validated or observed for changes.
 - `maxLength(n)`
 - `number`
 - `email`
+- `email_with_tld`
```

Comment:
I would prefer camel case

Reply:
Likewise for the other variables except if you have a reason as to why you chose it.